src/Result.php

```diff
@@ -72,6 +72,22 @@ public function getAttributeErrorObjects(string $attribute): array
         return $this->getAttributeErrorsMap($attribute, static fn (Error $error): Error => $error);
     }
 
+    /**
+     * @psalm-return array<int|string, non-empty-list<int|string>>
+     */
+    public function getTopLevelAttributeErrors(): array
+    {
+        $errors = [];
+        foreach ($this->errors as $error) {
+            $firstItem = $error->getValuePath()[0] ?? null;
+            if ($firstItem !== null) {
```

Comment:
Maybe add `null` errors with key `''` (empty string)?
This is will be consistently with `getErrorsIndexedByPath`.

Reply:
:heavy_check_mark: